Source/main.cpp

```diff
@@ -153,6 +153,7 @@ void main_main ()
     amrex::Print() << " exchange_coupling    = " << exchange_coupling   << "\n";
     amrex::Print() << " DMI_coupling         = " << DMI_coupling        << "\n";
     amrex::Print() << " anisotropy_coupling  = " << anisotropy_coupling << "\n";
+    amrex::Print() << " FFT_solver           = " << FFT_solver          << "\n";
```

Comment:
move this so it's right after demag_coupling, and only print this line if demag_coupling==1